From 6528b94ad7afb94a5e26f7376982917b7a407f76 Mon Sep 17 00:00:00 2001
From: Marco Chiotti <marcox.chiotti@intel.com>
Date: Thu, 31 Mar 2016 14:43:08 +0200
Subject: [PATCH 3/3] [BT] Remove wrong call to set_wake_locked()

bt_lpm_rfkill_set_power() is called before
bt_lpm.tty_dev is initialized so the calls to
set_wake_locked() shall be removed from it

Change-Id: I1d5ab82c60a97de8a0dae796788fb831580114f8
Signed-off-by: Marco Chiotti <marcox.chiotti@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-25998
Reviewed-on: https://android.intel.com:443/488867
---
 drivers/misc/bt_lpm/bt_lpm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/misc/bt_lpm/bt_lpm.c b/drivers/misc/bt_lpm/bt_lpm.c
index 8c21826..11d615b 100644
--- a/drivers/misc/bt_lpm/bt_lpm.c
+++ b/drivers/misc/bt_lpm/bt_lpm.c
@@ -222,7 +222,6 @@ static int bt_lpm_rfkill_set_power(void *data, bool blocked)
 		gpio_set_value(bt_lpm.gpio_reset, 1);
 #else
 		gpio_set_value(bt_lpm.gpio_enable_bt, 1);
-		set_wake_locked(1);
 
 #endif
 		pr_debug("%s: turn BT on\n", __func__);
@@ -232,7 +231,6 @@ static int bt_lpm_rfkill_set_power(void *data, bool blocked)
 		gpio_set_value(bt_lpm.gpio_reset, 0);
 #else
 		gpio_set_value(bt_lpm.gpio_enable_bt, 0);
-		set_wake_locked(0);
 #endif
 		pr_debug("%s: turn BT off\n", __func__);
 	}
-- 
1.9.1

